testsuite/tests/typing-misc/records.ml

```diff
@@ -248,12 +248,14 @@ Error: This variant or record definition does not match that of type d
        The types are not equal.
 |}]
 
-type unboxed = d = {x:float} [@@unboxed]
+type mono = {foo:int}
+type unboxed = mono = {foo:int} [@@unboxed]
```

Comment:
`includecore` changed to take into account the changes to `record_representation` and `variant_representation`, and one result is that in cases where there are multiple reasons to give an error, we might give a different one now.  The old test was kind of wild - the error complains about unboxed, but actually the record is totally different too, and now we get that error first.  Since the point of this test case was to test a mismatch on `@@unboxed`, I made the records more similar, so that we hit the unboxed error.